src/web/mod.rs

```diff
@@ -15,11 +15,11 @@ macro_rules! ctry {
             Err(error) => {
                 let request: &::iron::Request = $req;
 
+                // TODO this should be `capture_anyhow` if error is `anyhow::Error`
                 ::log::error!(
-                    "called ctry!() on an `Err` value: {:?}\nnote: while attempting to fetch the route {:?}\n{:?}",
+                    "called ctry!() on an `Err` value: {:?}\nnote: while attempting to fetch the route {:?}",
                     error,
                     request.url,
-                    ::backtrace::Backtrace::new(),
```

Comment:
Why remove this?

Reply:
it's the same backtrace as the one attached by sentry itself through `attach_stacktrace`. 

Of course we could decide to keep it only for the logs on the server

Reply:
with the new macros we will report the original error / its backtrace